tools/@aws-cdk/construct-metadata-updater/lib/metadata-updater.ts

```diff
@@ -0,0 +1,500 @@
+import { ClassDeclaration, Project, QuoteKind, SourceFile, Symbol, SyntaxKind } from "ts-morph";
+import * as path from "path";
+import * as fs from "fs";
+// import SyntaxKind = ts.SyntaxKind;
+
+const DIRECTORIES_TO_SKIP = [
+  "node_modules",
+  "dist",
+  "build",
+  "decdk",
+  "awslint",
+  "test",
+  "aws-eks-v2-alpha"
```

Comment:
why do you skip eks v2, and if this is a temp change, can you add TODO comment here, and also in the EKS alpha module to let us know when we should remove this line

Reply:
When I made the changes, eks-v2-alpha was marked as private. It seems that it's now public so we should be able to add it now. Let me remove this check.